.gitignore

```diff
@@ -6,3 +6,4 @@ temp
 explorations
 TODOs.md
 *.log
+.vscode
```

Comment:
This is unnecessary. It's intentionally kept in there to ensure consistent TS version is used.